l10n_it_intrastat_statement/report/report_intrastat_mod1.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_intrastat_mod1">
+        <t t-call="web.html_container">
+            <t t-foreach="docs" t-as="o">
+                <t t-call="web.internal_layout">
+                    <div class="page">
+                        <div class="row">
+                            <div class="col-3 offset-9">
+                                <table class="table table-bordered">
+                                    <tr>
+                                        <td>Mod.INTRA-1</td>
+                                    </tr>
+                                </table>
+                            </div>
+                            <div class="offset-4">
+                                <table class="table table-bordered">
+                                <tr>
+                                    <td><b>ELENCO RIEPILOGATIVO DELLE CESSIONI
+                                    <br/>INTRACOMUNITARIE DI BENI E DEI SERVIZI RESI</b>
+                                    </td>
+                                </tr>
+                                </table>
+                            </div>
+                        </div>
+                        <hr />
+                        <table class="table table-bordered">
+                            <th colspan="3" class="fondo">PERIODICITÀ ELENCO</th>
+                            <tr>
+                                <td width="20%" rowspan="2">PERIODO DI RIFERIMENTO</td>
+                                <td>MESE: <span class="font_bold" t-esc="o.period_number"/> ANNO: <span class="font_bold" t-esc="o.fiscalyear"/></td>
+                                <td class="center" width="20%" rowspan="2"> NUMERO DI RIFERIMENTO <br/><span class="font_bold" t-esc="o.sale_statement_sequence"/></td>
+                            </tr>
+                            <tr>
+                                <td>TRIMESTRE</td>
```

Comment:
Bisogna valorizzare "trimestre" invece di "mese" quando il tipo di periodo è "trimestre"

Reply:
Inoltre, manca la dicitura "Le informazioni delle sezioni..." che è invece presente nel doc https://www.adm.gov.it/portale/documents/20182/908833/AllegatoI.pdf/f80736f8-d0b5-45ee-9081-34505fdc610c

Reply:
Sistemate le stampe mod.1 e mod.2 (valorizzato TRIMESTRE o MESE), inserito nuovo logo (in tutte e 4 le stampe) e spazio per data e firma in fondo per mod.1 e mod.2 (come presenti nel link); sistemato il font per farle stare su una pagina.

Modificati i due campi data.